packages/wdio-local-runner/src/index.js

```diff
@@ -14,37 +14,47 @@ export default class LocalRunner extends EventEmitter {
         this.config = config
     }
 
+    /**
+     * nothing to initialise when running locally
+     */
     initialise () {}
 
-    run (options) {
-        const {
-            cid, command, configFile, argv, caps, processNumber,
-            specs, server, isMultiremote
-        } = options
-        const logFile = fs.createWriteStream(path.join(this.config.logDir, `wdio-${cid}.log`))
+    run ({ cid, command, configFile, argv, caps, processNumber, specs, server, isMultiremote }) {
         const runnerEnv = Object.assign(this.config.runnerEnv, {
```

Comment:
Object.create instead of Object.assign seems to be what we want here (unchanged code so might be unrelated)